src/amo/components/AddonsByAuthorsCard/index.js

```diff
@@ -55,43 +54,13 @@ export class AddonsByAuthorsCardBase extends React.Component<Props> {
   }
 
   componentWillMount() {
```

Comment:
I think it would maybe make sense to always call `dispatchFetchAddonsByAuthors()` in `componentWillMount()`.